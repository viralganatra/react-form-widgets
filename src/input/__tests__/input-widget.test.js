import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import InputWidget from '../input-widget';

describe('InputWidget', () => {
    it('should render correctly with a default type of text if none supplied', () => {
        const InputDefault = shallow(<InputWidget />);
        const InputCustom = shallow(<InputWidget type="number" min="1" max="10" />);

        expect(shallowToJson(InputDefault)).toMatchSnapshot();
        expect(shallowToJson(InputCustom)).toMatchSnapshot();
    });
});
