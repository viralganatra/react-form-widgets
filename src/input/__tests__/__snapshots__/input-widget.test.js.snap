exports[`InputWidget should render correctly with a default type of text if none supplied 1`] = `
<input
  type="text" />
`;

exports[`InputWidget should render correctly with a default type of text if none supplied 2`] = `
<input
  max="10"
  min="1"
  type="number" />
`;
